src/main/java/com/group6/harmoniq/controllers/SpotifyController.java

```diff
@@ -295,4 +322,188 @@ private Track getTopTrack(String accessToken) throws Exception {
             throw new Exception("Failed to get top tracks", e);
         }
     }
+
+    private List<Artist> getTopArtists(String accessToken) throws Exception {
```

Comment:
The existing `getTopArtist` method can be modified to get the top 10 artists instead of creating a new method